ha/k8s_setup/ha_setup.py

```diff
@@ -266,7 +266,8 @@ def process(self):
 
             Log.info('Performing event_manager subscription')
             event_manager = EventManager.get_instance()
-            event_manager.subscribe(const.EVENT_COMPONENT, [SubscribeEvent(const.POD_EVENT, ["online", "failed"])])
+            event_manager.subscribe(const.EVENT_COMPONENT,
+                                    [SubscribeEvent(const.POD_EVENT, ["online", "failed"], ["server", "data"])])
```

Comment:
you can use enum class values here instead of hardcoded strings, so one can understand if any change is required tomorrow.

Reply:
Done.